libs/sdk-core/src/models.rs

```diff
@@ -839,6 +871,11 @@ pub struct ReceivePaymentResponse {
 pub struct SendPaymentRequest {
     /// The bolt11 invoice
     pub bolt11: String,
+    /// By default, if the LSP supports it, trampoline payments will be tried. Trampoline payments
+    /// outsource pathfinding to the LSP. Trampoline payments improve payment performance, but are
+    /// generally more expensive in terms of fees and they compromise on privacy. Setting this flag
+    /// to `true` will ensure no trampoline payment is attempted.
+    pub no_trampoline: bool,
```

Comment:
I think I would find `skip_trampoline` slightly more descriptive, but `no_trampoline` is also good.

Reply:
It's always hard to find a good name for a negative boolean. I considered skip_trampoline, but I found no_trampoline more descriptive and a little less difficult to understand.